hunts/src/puzzle-table.js

```diff
@@ -27,6 +28,12 @@ const TABLE_COLUMNS = [
     Cell: NameCell,
     className: "col-4",
   },
+  {
+    Header: "",
+    accessor: (row) => row.recent_editors,
```

Comment:
Is this for sorting by this column? Will this sort from most to least recent editors?

Also, will the search box allow you to filter puzzles by who is working on them? (say, I really want to puzzle with Joe)